h/util/user.py

```diff
@@ -23,3 +25,10 @@ def split_user(userid):
 
 def format_userid(username, authority):
     return f"acct:{username}@{authority}"
+
+
+def get_user_url(user, request: Request) -> str | None:
+    if user.authority == request.default_authority:
+        return request.route_url("stream.user_query", user=user.username)
```

Comment:
I reckon we discussed we would be [using the user activity route](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1739286195078829?thread_ts=1739263417.741729&cid=C1MA4E9B9), rather than "stream.user_query", even if the latter is an alias.